README.md

```diff
@@ -17,16 +17,71 @@ standard implementation of lightweight Virtual Machines (VMs) that feel and
 perform like containers, but provide the workload isolation and security
 advantages of VMs.
 
+## License
+
+The code is licensed under an Apache 2.0 license.
+See [the license file](LICENSE) for further details.
+
+## Platform support
+
+Kata Containers currently runs on 64-bit systems supporting the following
+technologies:
+
+| Architecture | Virtualization technology |
+|-|-|
+| `x86_64` / `amd64` | [Intel](https://www.intel.com) VT-x, AMD SVM |
+| `aarch64` ("`arm64`")| [ARM](https://www.arm.com) Hyp |
+| `ppc64le` | [IBM](https://www.ibm.com) Power |
+| `s390x` | [IBM](https://www.ibm.com) Z & `LinuxONE` SIE |
```

Comment:
"LinuxOne" has to be rendered in backticks to avoid failing the spell checker currently.

Reply:
https://github.com/kata-containers/tests/pull/4413 includes a fix for this.

Reply:
That PR has now landed so I've removed the backticks.